development.md

```diff
@@ -0,0 +1,18 @@
+## Indexing of AppMap Data
+
+### Indexed `appmap.yml` Files
+
+`appmap.yml` files, which are located in a project, are indexed.
```

Comment:
Thanks, and just to be clear, by `are indexed`, you mean they are indexed by the `appmap index` command?

Reply:
No, I meant "indexed by the IDE", i.e. `appmap.yml` files can be located quickly and I can quickly scan `.appmap.json` files.